### DISCUSSION TITLE
dil değiştirince puanlar görünmüyor

### DISCUSSION THREAD

**Discussion by gokhan85** [IBAX-io/go-ibax#453] [Ideas] [answered]
### Clear and concise description of the problem

sayfanın planın ingilizceden baska bir dile çevirdiğimiz zaman topladigimiz puanlar görünmüyor arkadaşlarımız çok soru soruyor bu konu hakkında. lütfen bu sorunu çözerseniz bilmeyen arkadaşlarımız soru sormak zorunda kalmaz.
When we translate the plan of the page from English to another language, the points we collect do not appear. Our friends ask a lot of questions about this subject. Please solve this problem, our friends who do not know do not have to ask questions.

### Suggested solution

bence bu sorun cözülebilir

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by Emircangzc** [comment]
This is a big problem

**Comment by gokhan85** [comment]
bunu çözer umarım ıbax görevlileri

**Comment by mackcarelock** [comment]
Benim de puanlarım eksik görünüyor wallet da 

Bir de RT lerim kabul edilmiyor ve puanlamıyor 

Ne yapacağım ?

**Comment by ibaxio** [comment, marked as answer]
This is due to display errors caused by automatic translation, please try to switch to English, we are on the way to support Turkish language.

**Reply by gokhan85** [reply]
umariz en kısa zamanda çözülür.anadili ingilizce olmayanlar için şu anda büyük bir sorun

**Comment by ghost** [comment]
Bu problem kısa zamanda ibax ekibi tarafından çözülür kesin olarak.

**Reply by ozgurizmir** [reply]
ekip çözer diye düşünüyorum
